website/docs/fun/counting_v2.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Counting System
+---
+
+This command controls the counting game and the statistics command.
+
+This version of a counting command allows for the usage of base 10 and roman numerals, optionally joined by the usage of math functions and/or operators to count from 1, incrementing by 1, until failure.
```

Comment:
```suggestion
A unique feature of this command is that, besides decimal numbers, it also accepts roman numerals and mathematical expressions as valid inputs. So if the next number is 2, `II` and `2*sin(pi/2)` would be considered correct as well.
```
I've not checked whether `2*sin(pi/2)` actually works with your code; replace with an appropriate example if not.

Reply:
Just checked, it does. The built-in calc command handles the calculation, the CC just converts values into an acceptable expression and invalidates if necessary.